README.md

```diff
@@ -4,6 +4,7 @@
 [![coveralls]](https://coveralls.io/github/Runnable/ponos?branch=master)
 [![dependencies]](https://david-dm.org/Runnable/ponos)
 [![devdependencies]](https://david-dm.org/Runnable/ponos#info=devDependencies)
+![Code Climate](https://codeclimate.com/github/Runnable/ponos/badges/gpa.svg)
```

Comment:
I don't mind having this, but use the pattern that's been set forth (link to the image at the bottom, named and linked here, and also use shields.io for the badge so it's consistent square styles)